### ISSUE TITLE
SUMMARY

### ISSUE THREAD

**Issue by gdedi001** [nhjortsvan/prj-rev-bwfs-dasmoto#1]
_Grade: Exceeds Expectations_

Great job on your project - I loved how you were able to incorporate the `<div>` tag making your HTML very organized! In addition to this, you utilized the `id` attribute properly which allowed you to target each one separately. These two aspects of your code helped you meet the design requirements to a tee! An area for improvement here is with your final div. Make sure you align opening and closing tags.

A recommendation here is to review the following article.. It will aid you as your projects get more complex. Read it thoroughly and view the examples! Keep up the great work!
[12 Principles for Clean Code](https://www.smashingmagazine.com/2008/11/12-principles-for-keeping-your-code-clean/)

**Comment by nhjortsvan** [comment]
Thank you Gabriel, I appreciate your suggestion and also the "12 Principles" reference.  This article makes a great case for designing class/id with reuse and change in mind.  Naming a class "red italic" can look silly after a CSS change to blue.  The analog in coding is the intelligent key (the key conveys meaning), which sounds cool, except when the context changes, then you are stuck with keys that don't "make sense".   I also subscribed to Smashing, thanks for the reference.  I'm glad to see your interest in VR.  When I was with ATT (then Pacific Bell in 1994), VPL was our customer.  I got to try out their Silicon Graphics-powered VR systems, using a Mattel Powerglove.   The shapes were just raw polygons, but it was jaw-dropping to be able to interact with my co-worker in this new world.
